tasks/setup-RedHat.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: "Install EPEL repo"
```

Comment:
I think I would set a condition on this so that if someone is already managing EPEL elsewhere they can skip the repo configuration here.

Reply:
Good point. I added a switch for determining whether to install EPEL.